File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaSucursal.java
package cuponsmart.modelo.pojo.respuesta;

import java.util.List;
import cuponsmart.modelo.pojo.entidad.Sucursal;

public class RespuestaSucursal extends Mensaje{
    private List<Sucursal> contenido;
    
    public List<Sucursal> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Sucursal> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaSucursal.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Sucursal

class RespuestaSucursal(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Sucursal>? = null
): Mensaje(error, mensaje)